backend/graphql/resolvers/postingResolvers.ts

```diff
@@ -12,8 +19,19 @@ const postingResolvers = {
     ): Promise<PostingResponseDTO> => {
       return postingService.getPosting(id);
     },
-    postings: async (): Promise<PostingResponseDTO[]> => {
-      return postingService.getPostings();
+    postings: async (
+      _parent: undefined,
+      {
+        closingDate,
+        statuses,
+        userId,
+      }: {
+        closingDate?: Date;
+        statuses?: [PostingStatus];
```

Comment:
```suggestion
        statuses?: PostingStatus[];
```

TypeScript differs from GraphQL in type annotation syntax, `[PostingStatus]` refers to a tuple of 1 element instead of an array